src/dmd/doc.d

```diff
@@ -2145,20 +2399,148 @@ extern (C++) bool isReservedName(const(char)* str, size_t len)
     return false;
 }
 
+/****************************************************
+* A delimiter for Markdown block quotes.
+*/
+struct MarkdownQuote
+{
+    int macroLevel;
+}
+
+/****************************************************
+* A delimiter for Markdown inline content like emphasis and links.
+*/
+struct MarkdownDelimiter
+{
+    size_t iStart;
+    char type;
+    int count;
+    int macroLevel;
+    bool leftFlanking;
+    bool rightFlanking;
```

Comment:
Ideally all members of a public date type should have a short Ddoc description.

Reply:
Also reverse sort the fields by size.